<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>327</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCvVideo" name="videoWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>200</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Frame:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="framesLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>0/0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Time:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="timesLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>00:00:00.000 / 00:00:00.000</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="playButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="icon">
         <iconset resource="qplay.qrc">
          <normaloff>:/pixmaps/play.png</normaloff>:/pixmaps/play.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSelectionSlider" name="frameSlider">
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="icon">
         <iconset resource="qplay.qrc">
          <normaloff>:/pixmaps/stop.png</normaloff>:/pixmaps/stop.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveFrameButton">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="icon">
         <iconset resource="qplay.qrc">
          <normaloff>:/pixmaps/picture_save.png</normaloff>:/pixmaps/picture_save.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>338</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSaveFrame"/>
    <addaction name="actionSaveVideo"/>
    <addaction name="actionInfo"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuMarkers">
     <property name="title">
      <string>Markers</string>
     </property>
     <addaction name="actionAddFrameMarker"/>
     <addaction name="actionEditMarkers"/>
    </widget>
    <widget class="QMenu" name="menuCut_List">
     <property name="title">
      <string>Cut List</string>
     </property>
     <addaction name="actionEnableSelection"/>
     <addaction name="actionCutSelection"/>
     <addaction name="actionEditCutList"/>
    </widget>
    <addaction name="actionFrameRate"/>
    <addaction name="menuMarkers"/>
    <addaction name="menuCut_List"/>
   </widget>
   <widget class="QMenu" name="menuFilters">
    <property name="title">
     <string>Filters</string>
    </property>
    <addaction name="actionFlipHorizontal"/>
    <addaction name="actionFlipVertical"/>
    <addaction name="actionRotate90cw"/>
    <addaction name="actionRotate90ccw"/>
    <addaction name="actionRotate180"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFilters"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionEditCutList">
   <property name="text">
    <string>Edit Cut List</string>
   </property>
  </action>
  <action name="actionSaveFrame">
   <property name="text">
    <string>Save Frame</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionEnableSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable Selection</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionCutSelection">
   <property name="text">
    <string>Cut Selection</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionSaveVideo">
   <property name="text">
    <string>Save Video</string>
   </property>
  </action>
  <action name="actionFlipVertical">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Flip Vertical</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionRotate90cw">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Rotate 90 CW</string>
   </property>
  </action>
  <action name="actionRotate90ccw">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Rotate 90 CCW</string>
   </property>
  </action>
  <action name="actionFlipHorizontal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Flip Horizontal</string>
   </property>
  </action>
  <action name="actionRotate180">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Rotate 180</string>
   </property>
  </action>
  <action name="actionFrameRate">
   <property name="text">
    <string>FrameRate</string>
   </property>
   <property name="toolTip">
    <string>Change Frame Rate</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAddFrameMarker">
   <property name="text">
    <string>Add Frame Marker</string>
   </property>
   <property name="toolTip">
    <string>Add Frame Marker</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionEditMarkers">
   <property name="text">
    <string>Edit Markers</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSelectionSlider</class>
   <extends>QSlider</extends>
   <header>QSelectionSlider.h</header>
  </customwidget>
  <customwidget>
   <class>QCvVideo</class>
   <extends>QWidget</extends>
   <header>QCvVideo.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qplay.qrc"/>
 </resources>
 <connections/>
</ui>
